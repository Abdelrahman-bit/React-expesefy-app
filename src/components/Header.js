import React from 'react';
import {NavLink} from 'react-router-dom'


const Header = () => (
    <header>
        <h1>Expencive</h1>
        <ul>
            <li><NavLink to="/" >Home</NavLink></li>
            <li><NavLink to="/creat">creat</NavLink></li>
            <li><NavLink to="/help">help</NavLink></li>
        </ul>
    </header>
)

export default Header;