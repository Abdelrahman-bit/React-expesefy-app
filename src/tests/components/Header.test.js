import React from 'react';
import { shallow } from 'enzyme';
import Header from '../../components/Header';

test('should render the header components correctly', () =>{
    const wrapper = shallow( <Header /> );
    expect(wrapper).toMatchSnapshot();
    // expect(wrapper.find('h1').text()).toBe('Expencive');
    // const renderer = new ReactShallowRenderer();
    // renderer.render( <Header /> );
    // expect( renderer.getRenderOutput()).toMatchSnapshot();
})