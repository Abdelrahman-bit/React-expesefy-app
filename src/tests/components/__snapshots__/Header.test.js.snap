// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the header components correctly 1`] = `
<header>
  <h1>
    Expencive
  </h1>
  <ul>
    <li>
      <NavLink
        to="/"
      >
        Home
      </NavLink>
    </li>
    <li>
      <NavLink
        to="/creat"
      >
        creat
      </NavLink>
    </li>
    <li>
      <NavLink
        to="/help"
      >
        help
      </NavLink>
    </li>
  </ul>
</header>
`;
